### PR TITLE
Add add command

### PR DESCRIPTION
**Pull request by KeithPJX**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -20,10 +20,15 @@
...
+import seedu.address.model.person.Gender;
 import seedu.address.model.person.Name;
 import seedu.address.model.person.Person;
 import seedu.address.model.person.Phone;
+import seedu.address.model.person.Race;
+import seedu.address.model.person.Religion;
+import seedu.address.model.person.Survey;
 import seedu.address.model.tag.Tag;
 
 /**
```
Comment (deepimpactmir) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:34]: Should probably change MESSAGE_USAGE too

File: src/main/java/seedu/address/logic/parser/CliSyntax.java [hunk 2]
```diff
@@ -12,4 +12,13 @@ public class CliSyntax {
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
 
+    public static final Prefix PREFIX_GENDER = new Prefix("g/");
```
Comment (deepimpactmir) [inline src/main/java/seedu/address/logic/parser/CliSyntax.java:14]: Cosmetic but we should just bunch them up with together with the other prefixes.

**Review by deepimpactmir — APPROVED** [review]
Looks good! :). Thanks for all the changes. Fixes #10

File: src/main/java/seedu/address/logic/commands/EditCommand.java (hunk 1 above)
Comment (KeithPJX) [reply src/main/java/seedu/address/logic/commands/EditCommand.java:34]: Will leave for the person doing the changes in the edit command.

**Comment by deepimpactmir** [comment]
Actually for the `VALIDATION_REGEX` in the new Race, Religion, Survey classes, is there any reason why `{Alnum}` is used? Should numbers be allowed in those attributes?

**Comment by KeithPJX** [comment]
I think alphanumeric for Survey is okay. I have changed the Race and Religion to only accept alphabetical characters in a PR that I have just opened.

**Pull request merged**